### DISCUSSION TITLE
DisableDefaultAssignment for key="phys:6" and mods = "CTRL|SHIFT"

### DISCUSSION THREAD

**Discussion by mrcjkb** [wezterm/wezterm#2821] [Q&A]
Hi.
First of all, thanks to wez for the great work. I'm really liking Wezterm so far!

I've been having a small issue with disabling the default keybinding for `ctrl` + `shift` + `6`.
In Neovim, `ctrl` + `^` is the keymap to switch to the alternate file. This is the same as `ctrl` + `shift` + `6` on my keyboard.

I have attempted to disable the wezterm keybinding by adding the following to my config:

```lua
keys = {
   {
	key = "phys:6",		
        mods = "CTRL|SHIFT",
	action = require('wezterm').action.DisableDefaultAssignment,
    },
},
```

But this doesn't seem to do anything.
When I open a bunch of tabs and start Neovim in one of them, `<C-^>` still switches to tab number 6.
Am I missing something?

**Comment by wez** [comment]
Did you try it using just `key="6"`?
You can use `wezterm show-keys --lua` to see what key assignments are in effect.

**Comment by mrcjkb** [comment]
Yes, I had tried that, and also `key=mapped:^`;
Thanks for the tip.

 The output did include
```lua
{ key = '6', mods = 'SHIFT|CTRL', action = act.ActivateTab(5) },
```

After swapping the order of the mods, that line was removed from the output:
```lua
{ key = '5', mods = 'SUPER', action = act.ActivateTab(4) },
{ key = '6', mods = 'SUPER', action = act.ActivateTab(5) },
{ key = '7', mods = 'SHIFT|CTRL', action = act.ActivateTab(6) },
{ key = '7', mods = 'SUPER', action = act.ActivateTab(6) },
{ key = '8', mods = 'SHIFT|CTRL', action = act.ActivateTab(7) },
```

But it still has the same behaviour. 

I just found out that I can also use `CTRL+6` to edit the alternate file in neovim. So for me, the problem is solved.

**Comment by mattcargile** [comment]
If someone comes back here, I had to do the below to make it work. I didn't want to get used to `<c-6>` because it isn't consistent cross-plat.
```lua
  { key = '6', mods = 'SHIFT|CTRL', action = wezterm.action.DisableDefaultAssignment },
  { key = '^', mods = 'SHIFT|CTRL', action = wezterm.action.DisableDefaultAssignment },
```
